nRF Toolbox/Utilities/CharacteristicReader.swift

```diff
@@ -28,11 +29,15 @@ enum ReservedFloatValues : UInt32 {
     case nres = 0x00800000
     case reserved = 0x00800001
     case negativeInfinity = 0x00800002
+    
+    static let firstReservedValue = ReservedFloatValues.positiveInfinity
 }
 
-let FIRST_S_RESERVED_VALUE = ReservedSFloatValues.positiveInfinity
-let FIRST_RESERVED_VALUE   = ReservedFloatValues.positiveInfinity
-let RESERVED_FLOAT_VALUES  = [Double.infinity, Double.nan, Double.nan, Double.nan, -Double.infinity]
+extension Double {
+    static var reservedValues: [Double] {
+        [Double.infinity, Double.nan, Double.nan, Double.nan, -Double.infinity]
+    }
```

Comment:
No need for `Double.` prefix in the values of the array since you already defined its type to be `[Double]`